README.md

```diff
@@ -1,6 +1,19 @@
 # lws
 
-LeaderWorkerSet: An API for deploying a group of pods as a unit of replication
+LeaderWorkerSet: An API for deploying a group of pods as a unit of replication. It aims to address common deployment patterns of AI/ML inference workloads, especially multi-host inference workloads where the LLM will be sharded and run across devices. 
+Design and proposal can be found here: http://bit.ly/k8s-LWS.
+
```

Comment:
Can we add the picture I posted in https://github.com/kubernetes-sigs/lws/pull/2#discussion_r1507919055? Perhaps under: "Conceptual view"

Reply:
updated